tex/cover.tex

```diff
@@ -4,9 +4,11 @@
 \begin{center}
 \oTUM{4cm}\\ 
 \vspace{5mm}     
-\LARGE FAKULT{\"A}T F{\"U}R INFORMATIK\\ 
-\vspace{0.5cm}
-\normalsize DER TECHNISCHEN UNIVERSIT{\"A}T M{\"U}NCHEN\\
+\LARGE \uppercase{School of Computation, Information and Technology}\\ 
+\vspace{0.2cm}
+\large \uppercase{Informatics}\\
```

Comment:
This line needs to go, as `Informatics` does not exist anymore. As we are waiting for official guidelines to emerge, I'd either
(a) only mention the School of CIT or
(b) add the Department as in the Header of the department website: https://www.cs.cit.tum.de/cs/startseite/

Reply:
This is how I'd interpret the guidelines (using a newline instead of a slash).
<img width="699" alt="Screenshot 2022-11-08 at 12 08 02" src="https://user-images.githubusercontent.com/102237363/200549169-17be48e6-d5eb-4444-bdee-4c8fbb0d5a07.png">

Happy to change it to "Department of Computer Science" if you think that's better. 🙂